app/components/gameStateManager.tsx

```diff
@@ -16,9 +17,9 @@ export const GameStateManager = ({
 }: GameStateProps) => {
   return (
     <div>
-      <button onClick={() => onGameStart(!gameState.start)}>
-        {gameState.start ? "resetart" : "start"}
-      </button>
+      <ReStartButton onClick={() => onGameStart(!gameState.start)}>
+        {gameState.start ? "Resetart" : "Start"}
```

Comment:
Ja tu zrobiłam literówkę. Powinno być Restart. Ja już to poprawiłam na swoim branchu. Ale to widzę, że też stylowany kompnent ma od tego nazwę. Ja bym to nazwała "GameManagerButton"